docs/Targets.md

```diff
@@ -0,0 +1,374 @@
+  - `"io.deis.oras.content.digest"`**(ORAS-specific)**: Represents the digest of the tar'ed content before compression.
+  - `"io.deis.oras.content.unpack"`**(ORAS-specific)**: A flag indicating that the blob represents a directory that needs to be decompressed and un-tar'ed.
+
+For example, consider the following directory structure on disk:
+
+```bash
+$ tree
+.
+├── hello.txt
+└── mydir
+    ├── bar.txt
+    └── foo.txt
+
+$ cat hello.txt
+hello
+
+$ cat mydir/foo.txt
+foo
+
+$ cat mydir/bar.txt
+bar
+```
+
+Adding the file `hello.txt` results in a blob with a descriptor similar to this:
+
+```json
+{
+  "mediaType": "application/vnd.custom",
+  "digest": "sha256:5891b5b522d5df086d0ff0b110fbd9d21bb4fc7163af34d08286a2e846f6be03",
+  "size": 6,
+  "annotations": {
+    "org.opencontainers.image.title": "hello.txt"
+  }
+}
+```
+
+Adding the directory `mydir` results in a tar-archived and compressed blob with a descriptor similar to this:
+
+```json
+{
+  "mediaType": "application/vnd.custom.tar+gzip",
+  "digest": "sha256:b14fa80f5afd3822dce52a711566586c8f89e5dc211e4b0d2819f219b102fe7a",
+  "size": 164,
+  "annotations": {
+    "io.deis.oras.content.digest": "sha256:9d418a0549e5bc45d195aacec889d30e520b9f2bd9feab57d57de6d9cdd66172",
+    "io.deis.oras.content.unpack": "true",
+    "org.opencontainers.image.title": "mydir"
+  }
+}
+```
+
+To create an artifact, a manifest needs to be [packed](https://pkg.go.dev/oras.land/oras-go/v2#PackManifest) to reference the two blobs and will also be stored in the file store. The manifest content might look like this:
+
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType": "application/vnd.example",
+  "config": {
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2,
+    "data": "e30="
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.custom",
+      "digest": "sha256:5891b5b522d5df086d0ff0b110fbd9d21bb4fc7163af34d08286a2e846f6be03",
+      "size": 6,
+      "annotations": {
+        "org.opencontainers.image.title": "hello.txt"
+      }
+    },
+    {
+      "mediaType": "application/vnd.custom.tar+gzip",
+      "digest": "sha256:b14fa80f5afd3822dce52a711566586c8f89e5dc211e4b0d2819f219b102fe7a",
+      "size": 164,
+      "annotations": {
+        "io.deis.oras.content.digest": "sha256:9d418a0549e5bc45d195aacec889d30e520b9f2bd9feab57d57de6d9cdd66172",
+        "io.deis.oras.content.unpack": "true",
+        "org.opencontainers.image.title": "mydir"
+      }
+    }
+  ],
+  "annotations": {
+    "org.opencontainers.image.created": "2025-03-07T08:34:23Z"
+  }
+}
+```
+
+In the file store, blobs with names are location-addressed by file paths, while other content (e.g., manifests and config blobs) is maintained in fallback storage. By default, the fallback storage is a limited in-memory CAS store.
```

Comment:
```suggestion
In the file store, blobs with names are location-addressed by file paths, while other content (e.g., manifests or config blobs) is maintained in fallback storage. By default, the fallback storage is a limited in-memory CAS store.
```
If there are no other content types other than manifests and config blobs, this should be an i.e.

Reply:
There could be unnamed layer blobs